http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -648,4 +691,83 @@ public String getTokenSignatureAlgorithm() {
         return tokenSignatureAlgorithm;
     }
 
+    public String getAuthenticationRequestFormat() {
+        return authenticationRequestFormat;
+    }
+
+    public void setAuthenticationRequestFormat(String authenticationRequestFormat ) {
+        this.authenticationRequestFormat = authenticationRequestFormat;
+    }
+
+    public String getRequestSignatureAlgorithm() {
+        return requestSignatureAlgorithm;
+    }
+
+    public void setRequestSignatureAlgorithm(String requestSignatureAlgorithm) {
+        this.requestSignatureAlgorithm = requestSignatureAlgorithm;
+    }
+
+    public String getRequestEncryptAlgorithm() {
+        return requestEncryptAlgorithm;
+    }
+
+    public void setRequestEncryptAlgorithm(String requestEncryptAlgorithm) {
+        this.requestEncryptAlgorithm = requestEncryptAlgorithm;
+    }
+
+    public String getRequestContentEncryptionMethod() {
+        return requestContentEncryptionMethod;
+    }
+
+    public void setRequestContentEncryptionMethod(String requestContentEncryptionMethod) {
+        this.requestContentEncryptionMethod = requestContentEncryptionMethod;
+    }
+
+    public String getRequestObjectSigningKeystoreFile() {
+        return requestObjectSigningKeystoreFile;
+    }
+
+    public void setRequestObjectSigningKeystoreFile(String keyStoreFile) {
+        this.requestObjectSigningKeystoreFile = keyStoreFile;
+    }
+
+    public String getRequestObjectSigningKeyStorePassword() {
```

Comment:
We should be consistent with Keystore or KeyStore in the method names in this file.